Tasks/AzureFunctionAppV1/deploymentProvider/BuiltInLinuxWebAppDeploymentProvider.ts

```diff
@@ -9,6 +9,7 @@ var zipUtility = require('azurermdeploycommon/webdeployment-common/ziputility.js
 
 const linuxFunctionStorageSetting: string = '-WEBSITES_ENABLE_APP_SERVICE_STORAGE true';
 const linuxFunctionRuntimeSettingName: string = '-FUNCTIONS_WORKER_RUNTIME ';
+const premiumPlanRunsFromPackage: string = ' -WEBSITE_RUN_FROM_PACKAGE true';
```

Comment:
Is this relevant for other type of Apps or this is specific to FunctionApp?

Reply:
This is specific to function app